### ISSUE TITLE
Hello, is the code you uploaded complete?

### ISSUE THREAD

**Issue by maming109** [d-ailin/CLIP-Guided-Decoding#6]
Hello, is the code you uploaded complete?
best whishes！

**Comment by d-ailin** [comment]
Yes, it is complete to run the experiments and the evaluation. Please let me know if you have any issues. Thanks!

**Comment by maming109** [comment]
> Yes, it is complete to run the experiments and the evaluation. Please let me know if you have any issues. Thanks!

Ok,Thank you,I do not have any issues.
